c7n/resources/hsm.py

```diff
@@ -110,3 +111,83 @@ class resource_type(TypeInfo):
         detail_spec = ('describe_luna_client', 'ClientArn', None, None)
         arn = id = 'ClientArn'
         name = 'Label'
+
+
+class DescribeCloudHSMBackup(DescribeSource):
+
+    def augment(self, resources):
+        for r in resources:
+            r['Tags'] = r.pop('TagList', ())
+        return resources
+
+    def resources(self, query):
+        resources = self.query.filter(self.manager, **query)
+        return [r for r in resources if r['BackupState'] != 'PENDING_DELETION']
```

Comment:
Wouldn't it be better to do server side filtering instead of client side?

Reply:
Oh, I read the above comments. Looks like there's a bug that's preventing us from supplying multiple states